client/assets/stylesheets/README.md

```diff
@@ -1,12 +1,7 @@
-Styles
-======
+# Styles
 
-Calypso uses Sass to build its CSS and compile it to `public/style.css`. Remember that all styles end up in a single file, so all styles will apply to every page, all the time. Make sure you namespace your styles for the page you are working on.
+Calypso uses Sass to build its CSS. Remember that all styles, even when loaded at different times, eventually end up on one page as part of a single HTML document. Make sure you namespace your styles for the page you are working on.
```

Comment:
I basically copied the changed part from https://github.com/Automattic/wp-calypso/pull/35779/files#diff-40cd1fdb84e4a1a41de351ebd5c8a86aR95